utopia-remix/server.js

```diff
@@ -81,8 +81,12 @@ installGlobals()
 async function createDevRequestHandler(initialBuild) {
   let build = initialBuild
   async function handleServerUpdate() {
+    // 0. give the rebuild time to breathe
+    await new Promise((res) => setTimeout(res, 1000))
```

Comment:
is there something we could explicitly await? I am always afraid of these fixed timeouts